optuna/integration/sklearn.py

```diff
@@ -56,6 +57,25 @@
 _logger = logging.get_logger(__name__)
 
 
+def _check_fit_params(
+    X,  # type: TwoDimArrayLikeType
+    fit_params,  # type: Dict
+    indices  # type: OneDimArrayLikeType
+):
+    # type: (...) -> Dict
+
+    if _sklearn_version >= '0.22.1':
+        return validation._check_fit_params(X, fit_params, indices)
+    else:  # '_sklearn_version < 0.22.1'
+        return {
+            key: validation._index_param_value(
```

Comment:
The symbol `_index_param_value` must be explicitly imported, i.e. `from sklearn.model_selection._validation import _index_param_value`. You could do a conditional import at the submodule scope at the top of this script.

Reply:
57e8e61 meets your intention?